docs/manage/reference/accounts.md

```diff
@@ -0,0 +1,45 @@
+---
+linkTitle: "Account Management"
+title: "Account Management"
+weight: 1000
+layout: "docs"
+type: "docs"
+no_list: true
+description: "Log in and out of your Viam account."
+aliases:
+  - /fleet/account/
+  - /cloud/account/
+---
+
+The [Viam app](https://app.viam.com/) is a web UI for managing and building machines.
+
+## Create account and log In
+
+To get started on the Viam app, you must log in as an authorized user.
+Viam supports sign up using Google, GitHub, Apple, and email.
+
+Navigate to [the main page](https://app.viam.com/).
+If you haven't created an account yet, click **Sign Up** to create a new account using your preferred Single Sign On method or your email address and a password.
+If you already have an account, click **Log In** to log in using your Single Sign On credentials or your email address and password.
+
+If you forget your password to the app, click **Forgot password** and enter your email address to obtain instructions to reset your password.
+
+{{< alert title="Info" color="info" >}}
+Accounts created from separate authentication sources are unique to each other.
+{{< /alert >}}
+
+<br>
+
+## Sign out
+
+To log out or sign out of the [Viam app](https://app.viam.com/), click on your profile icon in the upper right corner of your browser window.
```

Comment:
```suggestion
To log out of the [Viam app](https://app.viam.com/), click on your profile icon in the upper right corner of your browser window.
```
redundant? but I like keeping at least one instance of "log out" since that's what I'd command F for